resilience4j/datadog_checks/resilience4j/check.py

```diff
@@ -0,0 +1,39 @@
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheck
+
+from .metrics import METRIC_MAP
+
+
+class Resilience4jCheck(OpenMetricsBaseCheck):
+    DEFAULT_METRIC_LIMIT = 0
+
+    def __init__(self, name, init_config, instances):
+        default_instances = {
+            'resilience4j': {
+                'metrics': [METRIC_MAP],
+                'send_distribution_sums_as_monotonic': 'true',
+                'send_distribution_counts_as_monotonic': 'true',
+            }
+        }
+
+        super(Resilience4jCheck, self).__init__(
+            name, init_config, instances, default_instances=default_instances, default_namespace='resilience4j'
+        )
+
+    def _http_check(self, url, check_name):
+        try:
+            response = self.http.get(url)
+            response.raise_for_status()
+        except Exception as e:
+            self.service_check(check_name, self.CRITICAL, message=str(e))
+        else:
+            if response.status_code == 200:
+                self.service_check(check_name, self.OK)
+            else:
+                self.service_check(check_name, self.WARNING)
+
+    def check(self, instance):
+        prometheus_url = instance.get("prometheus_url")
+        if prometheus_url is None:
+            raise ConfigurationError("Each instance must have a url to the metrics endpoint")
+
+        super(Resilience4jCheck, self).check(instance)
```

Comment:
```suggestion
    def check(self, _):
        super().check(_)
```
We recommend using the `openmetrics_endpoint` property instead of the old `prometheus_url`. However, this property is automatically picked by the OpenMetricsBaseCheckV2 class, so we don't need to parse it or validate it manually.